src/main/java/com/flansmod/common/guns/GunType.java

```diff
@@ -14,6 +16,7 @@
 import net.minecraft.item.ItemStack;
 import net.minecraft.nbt.NBTTagCompound;
 import net.minecraft.nbt.NBTTagList;
+import org.classpath.icedtea.Config;
```

Comment:
What is this for? I'm unable to compile the project due to this reference here. It's present in GunType, ShootableType, PartType, and ToolType.

Reply:
Ahhh... Rip I missed that sorry. I think that's something IntelliJ inserts when you type in Config instead of ConfigMap. That can definitely be removed.